test/libsolidity/syntaxTests/inheritance/override/interface_exception/abstract_needed.sol

```diff
@@ -0,0 +1,7 @@
+abstract contract I {
+	function f() external virtual;
+}
+contract C is I {
+	function f() external override {}
```

Comment:
The test is only meaningful if you do not have `override` here, isn't it?

Reply:
We aren't checking for abstract, only for interface though?